src/presenter/new-point-presenter.js

```diff
@@ -0,0 +1,94 @@
+import { UpdateType, UserAction } from '../const.js';
+import { RenderPosition } from '../framework/render.js';
+import EventEdit from '../view/event-edit.js';
+import { render, remove } from '../framework/render.js';
+
+export default class NewPointPresenter {
+  #pointListContainer = null;
+  #handlePointChange = null;
+  #handleDestroy = null;
+  #destinationsModel = null;
+  #offersMolel = null;
+
+  #pointEditComponent = null;
+
+  constructor({pointListContainer, offersModel, destinationsModel, onPointChange, onDestroy}) {
+    this.#pointListContainer = pointListContainer;
+
+    this.#offersMolel = offersModel.offers;
```

Comment:
Записываем модель, не данные. Данные ещё пустые, они не загружались ещё с сервера
```suggestion
    this.#offersModel = offersModel;
```